### DISCUSSION TITLE
Can we use different k3d version?

### DISCUSSION THREAD

**Discussion by gajenc** [egovernments/DIGIT-DevOps#815] [Q&A]
I have higher or lower version of the k3d version, can I still use the setup command mentioned in the DIGIT Quickstart setup guide

**Comment by gajenc** [comment]
Yes, you should be able to use different versions of the k3d, however there could be a slight changes in the command argument and syntax, which you may have to customize accordingly

**Reply by shashwat-egov** [reply]
@gajendran-egov As a matter of fact, I was installing k3d on my windows machine today. I used the command given in the quickstart guide - `choco install k3d`. This installs the latest version of k3d i.e. 5.4.0. Then, I tried creating kubernetes cluster by running this as per the quickstart guide - `k3d cluster create --k3s-server-arg "--no-deploy=traefik" --agents 2 -v "/kube:/kube@agent[0,1]" -v "/kube:/kube@server[0]" --port "80:80@loadbalancer"`.  Apparently, 5.4.0 version of k3d does not recognize `--k3s-server-arg` flag. I uninstalled k3d@5.4.0 and installed k3d@4.0.0 which resolved this issue. So, it'll be helpful to maybe update the quickstart guide command to - `choco install k3d --version 4.0.0`. This way, the quickstart documentation will be a little more beginner friendly and people will get DIGIT up and running faster on their windows machine without having to google/come back to us with queries like why this flag is not supported or how they can go about installing a specific version of k3d etc.
